roles/pulp-content/README.md

```diff
@@ -1,12 +1,10 @@
 Pulp Content
 ============
 
-A role to setup Pulp 3's content serving service.
+A role to setup Pulp 3's content serving:
```

Comment:
A role to setup content serving in Pulp 3 or A role to setup serving of the content in Pulp 3